src/core/createConfigValidators.js

```diff
@@ -38,5 +38,17 @@ export default () => ({
   onBeforeEventSend: {
     defaultValue: () => undefined,
     validate: callback()
+  },
+  datasetId: {
+    defaultValue: undefined,
```

Comment:
I agree that it is weird.  If there is no defaultValue, then the field is required.  We should probably have a `required: false` option instead of requiring an undefined defaultValue.

Reply:
100%.